host/xtest/regression_1000.c

```diff
@@ -841,7 +841,9 @@ static void xtest_tee_test_1009(ADBG_Case_t *c)
 {
 	xtest_tee_test_1009_subcase(c, "TEE Wait 0.1s", 100, false);
 	xtest_tee_test_1009_subcase(c, "TEE Wait 0.5s", 500, false);
+#ifndef CFG_CRYPTO_SE05X
```

Comment:
How is this related to SE05X? You mention "SE05x runs single threaded", do you mean `xtest` doesn't use a compliant pthread library on the platform you're testing?

Reply:
sorry no I think I didnt document this anywhere: to run the SE05X, we configure CFG_NUM_THREADS=1 which will cause this particular test to fail.

With that we avoid having to synchronize access to the SE05x: ie, when OP-TEE has sent RPC to Linux with an APDU frame and is waiting for a response, we cant allow for any other request to go through...plug and trust will have to get a response first. So at the moment we have this limitation. I dont know, maybe it doesn't  quite justify the conditional compile on this regression_1000? or perhaps the test could check whether OP-TEE has been compiled with just one thread?

Reply:
So instead of using a mutex you configure the system with a single thread?

Reply:
Not the system, OP-TEE. 
And yes, but of course, since the option to do so is presented to the user. 
If that is an issue, then you just remove the option - which incidentally also breaks this test-  and I will prioritize the scheduling among my other deliverables...time is a limited resource as I am sure you know and functionality came before performance in this case.

Reply:
btw there is a bit of debugging and a lot of concurrency testing involved in implementing this change to the Plug-and-Trust repository (I am hoping is not a OP-TEE change)..wont be nothing major but yes, I have been postponing it

Reply:
We don't. Being GP compliant is a high priority in OP-TEE.

Reply:
That said, I guess we can disable this test when CFG_NUM_THREADS=1.

Reply:
@jenswi-linaro finally got around to addressing the thread issue in the plug and trust. So removing the need for the SE050.